### PR TITLE
Feature1 - Display Wedding Styles

### LINKED ISSUE #8: Feature 1: Display Wedding Styles

### PR DESCRIPTION
**Pull request by Pritha-crypto**
- [S ] Display Wedding Styles Added

### CODE REVIEW

**Review by Pritha-crypto — COMMENTED** [review]
Html code for list display wedding style is added

File: src/main/webapp/listweddingstyles.jsp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<%@page import="java.util.Set"%>
+<%@page import="java.util.Map"%>
+
+<%@page import="java.util.List"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/listweddingstyles.jsp:4]: Removed unused import

File: src/main/webapp/listweddingstyles.jsp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<%@page import="in.pritha.service.WeddingStyleDisplay"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/listweddingstyles.jsp:9]: Add lang attribute. 
<html lang="en">

File: src/main/webapp/listweddingstyles.jsp [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	<main class="container-fluid">
+	<br><br/>
+	<h3 id="heading">Wedding Styles</h3>
+	<br><br/>
+	<table class="table table-bordered">
+	<thead>
+	<tr><th>S.no</th><th>Wedding Styles</th><th>Package(Rs)</th></tr>
+	</thead>
+	<tbody>
+	<%-- <% Scriplets---> to write java code%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/listweddingstyles.jsp:28]: Remove not relevant comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**